### DISCUSSION TITLE
Serverpod 1.2 can't create a project

### DISCUSSION THREAD

**Discussion by Disregard-Therest** [serverpod/serverpod#1691] [Q&A] [answered]
Hi, the initial script can't finish

Last line still in progress after 20 minutes. I think it is not real and there must be some error.

![image](https://github.com/serverpod/serverpod/assets/98102396/471f71b5-529b-4f65-8a3a-bb51b7addf94)

**Comment by Isakdl** [comment]
Unless you have a super slow internet connection it does seem like there is an error here. 

Could you run the command with `-v` ? and print the output here.

**Comment by danny-alder** [comment, marked as answer]
Don't know if it's the exact same issue but I had a similar issue where this would happen if I had already created a project with the same name before. I solved the issue by going into Docker and deleting the old Containers, Images and Volumes that the original project had created/downloaded. Once I had done that then running `serverpod create` worked and would no longer get stuck on that step.

**Reply by Isakdl** [reply]
Right, yes this is true, we made a fix for this, not sure if that was shipped in rc 1, probably not, but it is merged too main. Now the command should timeout if running for too long, and secondly for the situation where the container already exists we give an explicit error message about that.

**Reply by Disregard-Therest** [reply]
Thank you, I didn't know about Images and Volumes, I cleared containers only))
